src/test/java/uk/gov/digital/ho/hocs/workflow/processes/FOI.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.digital.ho.hocs.workflow.processes;
+
+import org.camunda.bpm.engine.test.Deployment;
+import org.camunda.bpm.engine.test.ProcessEngineRule;
+import org.camunda.bpm.engine.test.mock.Mocks;
+import org.camunda.bpm.extension.process_test_coverage.junit.rules.TestCoverageProcessEngineRule;
+import org.camunda.bpm.extension.process_test_coverage.junit.rules.TestCoverageProcessEngineRuleBuilder;
+import org.camunda.bpm.scenario.ProcessScenario;
+import org.camunda.bpm.scenario.Scenario;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.digital.ho.hocs.workflow.BpmnService;
+
+import static org.camunda.bpm.engine.test.assertions.ProcessEngineTests.withVariables;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@Deployment(resources = "processes/FOI.bpmn")
+public class FOI {
+
+    @Rule
+    @ClassRule
+    public static TestCoverageProcessEngineRule rule = TestCoverageProcessEngineRuleBuilder.create()
+            .assertClassCoverageAtLeast(1.0)
+            .build();
+
+    @Rule
+    public ProcessEngineRule processEngineRule = new ProcessEngineRule();
+    @Mock
+    BpmnService bpmnService;
+    @Mock
+    private ProcessScenario FOIProcess;
+
+    @Before
+    public void defaultScenario() {
+
+        Mocks.register("bpmnService", bpmnService);
+
+    }
+
+    @Test
+    public void happyPath() {
+
+        Scenario.run(FOIProcess)
+                .startByKey("FOI")
+                .execute();
+
+        verify(FOIProcess)
+                .hasCompleted("Activity_1d2ue6g");
```

Comment:
Can we use a meaningful name instead of Activity_1d2ue6g ?

Reply:
That's an internal id used by camunda, it is given a proper real name but the tests use the internal id.